sps20/content/venue/contents.lr

```diff
@@ -1,38 +1,51 @@
-title: Venue
+title: Location
 ---
-description: A guide to the venue
+description: A guide to the conference venue
 ---
 ordering: 1
 ---
 body:
 
-# Venue
-The Swiss Python Summit is held on the campus of the **[OST Eastern Switzerland University of Applied Sciences](https://www.ost.ch/)** (formerly HSR) in **[Rapperswil](https://www.instagram.com/visitrapperswil/)** at the **Aula** in **Building 4** (upstairs).
+# Conference Venue
 
 ## Location
-<div class="media map" data-lat="47.223307" data-lon="8.816796" data-zoom="9" data-title="OST University of Applied Sciences">&nbsp;</div>
 
-## Conference Room
-![aula](./aula.jpg)
-
-The conference will take place in the **Aula (building 4)** upstairs. Or just follow the signs on campus.
-## Quiet Room
+The Swiss Python Summit is held on the campus of the OST Campus (formerly HSR) in Rapperswil.
 
-There is a separate room for working or for relaxing. Free wifi as well as power connections are available
+**Address:**  
```

Comment:
```suggestion
### Address
```

Or do 3rd level headers look strange? Same for the others.

Reply:
Done